lib/ember/handlebars/helper.rb

```diff
@@ -44,6 +44,10 @@ def template_path(path, config)
 
         path = path.split('/')
 
+        # Strip the '.raw' suffix if there is one.
```

Comment:
When https://github.com/tricknotes/ember-handlebars-template/pull/19/files#r52170894 is done, these lines are no longer necessary.